spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "spec_helper"
+require_relative "web_helpers"
+
+feature "log into chitter" do
+
+  scenario "log into chitter" do
+    user = User.new(email: "mariann013@email.com", username: "mariann013", password: "Password")
```

Comment:
Metrics/LineLength: Line is too long. [96/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)